### PR TITLE
Fix Failing Tests

### PR DESCRIPTION
**Pull request by muhammad-ammar**

### CODE REVIEW

**Comment by muhammad-ammar** [comment]
@symbolist Can you please review?

File: openassessment/xblock/test/data/save_scenario.xml [hunk 1]
```diff
@@ -85,8 +85,8 @@
     </rubric>
     <assessments>
         <assessment name="peer-assessment"
-          start="2014-12-20T19:00"
-          due="2014-12-21T22:22"
+          start="2016-12-20T19:00"
+          due="2016-12-21T22:22"
```
Comment (symbolist) [inline openassessment/xblock/test/data/save_scenario.xml]: Ha! Maybe you should make this 2020 or something?

**Comment by muhammad-ammar** [comment]
@symbolist Done

**Comment by symbolist** [comment]
:+1:

**Pull request merged**